typed-racket-test/fail/bad-struct-prop.rkt

```diff
@@ -0,0 +1,6 @@
+#;
+(exn-pred " Type Checker: type mismatch")
+#lang typed/racket
+(struct foo ([x : Number]) #:property prop:custom-write
+  (lambda ([n : Number] [p : Output-Port] [b : Boolean]) : Void
```

Comment:
This type annotation is wrong. The third argument `b` (the "mode" argument) should have the type `(U Boolean 0 1)`, not `Boolean`.